Switch to upstream PyTorch, deprecate IPEX dependency

As the Intel XPU has been upstream to the public torch, we are going to remove the IPEX dependency of the Triton for JIT compiling.
Need to clean up all the `import intel_extension_for_pytorch` in the Triton XPU code.

The related sub issues for this task:

- https://github.com/intel/intel-xpu-backend-for-triton/issues/770

Last nights failed run w/ https://github.com/pytorch/pytorch/commit/f9dae86222aaf15ea085c7774da70781bae46ff9: 
```
=========================== short test summary info ============================
FAILED language/test_core.py::test_reduce1d[64-16-1-argmax-tie-break-left-int8-512] - OverflowError: Python integer 432 out of bounds for int8
FAILED language/test_core.py::test_abs_fp8[in_dtype2] - NotImplementedError: The operator 'aten::__lshift__.Scalar' is not currently implemented for the XPU device. Please open a feature on https://github.com/intel/torch-xpu-ops/issues. You can set the environment variable `PYTORCH_ENABLE_XPU_FALLBACK=1` to use the CPU implementation as a fallback for XPU unimplemented operators. WARNING: this will bring unexpected performance compared with running natively on XPU.
FAILED language/test_core.py::test_reduce1d[4-32-1-argmax-tie-break-left-int8-512] - OverflowError: Python integer 432 out of bounds for int8
FAILED language/test_core.py::test_full[shape0-uint16] - RuntimeError: "eq_xpu" not implemented for 'UInt16'
FAILED language/test_core.py::test_full[shape0-uint32] - RuntimeError: "eq_xpu" not implemented for 'UInt32'
FAILED language/test_core.py::test_full[shape0-uint64] - RuntimeError: "eq_xpu" not implemented for 'UInt64'
FAILED language/test_core.py::test_full[shape1-uint16] - RuntimeError: "eq_xpu" not implemented for 'UInt16'
FAILED language/test_core.py::test_full[shape1-uint32] - RuntimeError: "eq_xpu" not implemented for 'UInt32'
FAILED language/test_core.py::test_full[shape1-uint64] - RuntimeError: "eq_xpu" not implemented for 'UInt64'
FAILED language/test_core.py::test_full[shape2-uint16] - RuntimeError: "eq_xpu" not implemented for 'UInt16'
FAILED language/test_core.py::test_full[shape2-uint32] - RuntimeError: "eq_xpu" not implemented for 'UInt32'
FAILED language/test_core.py::test_full[shape2-uint64] - RuntimeError: "eq_xpu" not implemented for 'UInt64'
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-True-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-True-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-2-True-True-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-True-none-tf32-float16-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-2-True-True-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-True-none-tf32-float16-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-True-none-tf32-float32-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-True-none-tf32-float32-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-False-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-False-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-2-True-True-none-tf32-float16-float16-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_random.py::test_randint[4,53-67830198458-int32-True] - OverflowError: Python integer 67830198458 out of bounds for uint32
FAILED language/test_random.py::test_randint[4,53-67830198458-int32-False] - OverflowError: Python integer 67830198458 out of bounds for uint32
FAILED language/test_random.py::test_randint[10-67830198458-int32-True] - OverflowError: Python integer 67830198458 out of bounds for uint32
FAILED language/test_random.py::test_randint[10-67830198458-int32-False] - OverflowError: Python integer 67830198458 out of bounds for uint32
FAILED language/test_random.py::test_randint[400-67830198458-int32-True] - OverflowError: Python integer 67830198458 out of bounds for uint32
FAILED language/test_random.py::test_randint[400-67830198458-int32-False] - OverflowError: Python integer 67830198458 out of bounds for uint32
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-False-none-tf32-float16-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-False-none-tf32-float16-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-False-none-tf32-float32-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-True-False-none-tf32-float32-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-False-True-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-False-True-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-2-True-True-none-tf32-float16-float16-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-False-True-none-tf32-float16-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-2-True-True-none-tf32-float16-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-False-True-none-tf32-float16-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-4-False-True-none-tf32-float32-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-2-False-False-none-tf32-float16-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-2-False-False-none-tf32-float16-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-2-False-False-none-tf32-float32-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-128-128-64-2-False-False-none-tf32-float32-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-True-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-True-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-True-none-tf32-float16-float16-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-True-none-tf32-float16-float16-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-True-none-tf32-float16-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-True-none-tf32-float16-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-True-none-tf32-float32-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-True-none-tf32-float32-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-False-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-False-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-False-none-tf32-float16-float16-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-False-none-tf32-float16-float16-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-False-none-tf32-float16-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-False-none-tf32-float16-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-False-none-tf32-float32-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-True-False-none-tf32-float32-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-True-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-True-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-True-none-tf32-float16-float16-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-True-none-tf32-float16-float16-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-True-none-tf32-float16-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-True-none-tf32-float16-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-True-none-tf32-float32-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-True-none-tf32-float32-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-False-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-False-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-False-none-tf32-float16-float16-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-False-none-tf32-float16-float16-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-False-none-tf32-float16-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-False-none-tf32-float16-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-False-none-tf32-float32-float32-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-2-False-False-none-tf32-float32-float32-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
```


With pytorch `153362fbc9e8642fb851a4de3b99e3871a2cc714` everything above passes except some `test_dot` failures. If I patch the LTS bug (where I currently have pytorch setup, see #1329) then I get 10 failures with precision issues.

```
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-trans-tf32-float32-float32-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.001

Mismatched elements: 251 / 256 (98%)
Max absolute difference: 0.12260317
Max relative difference: inf
 x: array([[-0.069057, -0.069733, -0.068503, -0.010018,  0.005891,  0.006578,
         0.102589,  0.0102  ,  0.005469,  0.018078,  0.028118, -0.001133,
         0.069394,  0.000721, -0.026283,  0.024772],...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-trans-tf32x3-float32-float32-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.001

Mismatched elements: 251 / 256 (98%)
Max absolute difference: 0.12267384
Max relative difference: inf
 x: array([[-0.069102, -0.069795, -0.068567, -0.010019,  0.0059  ,  0.006594,
         0.102642,  0.010193,  0.005472,  0.018109,  0.028152, -0.001136,
         0.069449,  0.000702, -0.026297,  0.024813],...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-trans-ieee-float16-float16-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.01

Mismatched elements: 215 / 256 (84%)
Max absolute difference: 0.1226
Max relative difference: inf
 x: array([[-0.0691  , -0.0697  , -0.06854 , -0.009995,  0.005913,  0.00658 ,
         0.1026  ,  0.01017 ,  0.00546 ,  0.0181  ,  0.02812 , -0.001139,
         0.06946 ,  0.000687, -0.02626 ,  0.02478 ],...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-trans-ieee-float16-float32-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.001

Mismatched elements: 251 / 256 (98%)
Max absolute difference: 0.1226
Max relative difference: inf
 x: array([[-0.0691  , -0.0697  , -0.06854 , -0.009995,  0.005913,  0.00658 ,
         0.1026  ,  0.01017 ,  0.00546 ,  0.0181  ,  0.02812 , -0.001139,
         0.06946 ,  0.000687, -0.02626 ,  0.02478 ],...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-trans-ieee-float32-float32-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.001

Mismatched elements: 251 / 256 (98%)
Max absolute difference: 0.12267384
Max relative difference: inf
 x: array([[-0.069102, -0.069795, -0.068567, -0.010019,  0.0059  ,  0.006594,
         0.102642,  0.010193,  0.005472,  0.018109,  0.028152, -0.001136,
         0.069449,  0.000702, -0.026297,  0.024813],...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-chain-dot-tf32-float32-float32-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.001

Mismatched elements: 254 / 256 (99.2%)
Max absolute difference: 0.7690249
Max relative difference: inf
 x: array([[-1.942635e-02, -2.114876e-01, -6.945647e-02, -1.716162e-01,
        -1.378054e-01, -1.165296e-01,  6.006219e-03,  1.291778e-01,
        -2.653402e-01, -1.434387e-01, -2.859902e-02,  5.359677e-03,...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-chain-dot-tf32x3-float32-float32-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.001

Mismatched elements: 255 / 256 (99.6%)
Max absolute difference: 0.76983273
Max relative difference: inf
 x: array([[-1.949987e-02, -2.117989e-01, -6.951781e-02, -1.716849e-01,
        -1.379933e-01, -1.167000e-01,  5.947299e-03,  1.293094e-01,
        -2.656509e-01, -1.435804e-01, -2.856388e-02,  5.384542e-03,...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-chain-dot-ieee-float16-float16-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.01

Mismatched elements: 243 / 256 (94.9%)
Max absolute difference: 0.7695
Max relative difference: inf
 x: array([[-1.9577e-02, -2.1179e-01, -6.9397e-02, -1.7163e-01, -1.3782e-01,
        -1.1664e-01,  6.0577e-03,  1.2939e-01, -2.6538e-01, -1.4355e-01,
        -2.8412e-02,  5.4092e-03,  4.2969e-02, -4.5715e-02,  3.4839e-01,...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-chain-dot-ieee-float16-float32-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.001

Mismatched elements: 254 / 256 (99.2%)
Max absolute difference: 0.7695
Max relative difference: inf
 x: array([[-1.9577e-02, -2.1179e-01, -6.9397e-02, -1.7163e-01, -1.3782e-01,
        -1.1664e-01,  6.0577e-03,  1.2939e-01, -2.6538e-01, -1.4355e-01,
        -2.8412e-02,  5.4092e-03,  4.2969e-02, -4.5715e-02,  3.4839e-01,...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
FAILED python/test/unit/language/test_core.py::test_dot[1-16-16-16-4-False-False-chain-dot-ieee-float32-float32-1] - AssertionError: 
Not equal to tolerance rtol=0.01, atol=0.001

Mismatched elements: 255 / 256 (99.6%)
Max absolute difference: 0.76983273
Max relative difference: inf
 x: array([[-1.949987e-02, -2.117989e-01, -6.951781e-02, -1.716849e-01,
        -1.379933e-01, -1.167000e-01,  5.947299e-03,  1.293094e-01,
        -2.656509e-01, -1.435804e-01, -2.856388e-02,  5.384542e-03,...
 y: array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],...
=========================================================================================================== 10 failed, 423 passed, 162 skipped, 10310 deselected in 227.53s (0:03:47) ===========================================================================================================
```

I am running unpatched on LTS now to see if the usual LTS failures occur, but it's pretty slow so I will let it run overnight. I did not see too many commits in PyTorch today, but let's see what tonight's run looks like.

@pbchekin I cannot reproduce the results from CI locally: 
https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/9740440080/job/26877561577
```
=========================== short test summary info ============================
FAILED language/test_core.py::test_abs_fp8[in_dtype2] - NotImplementedError: The operator 'aten::__lshift__.Scalar' is not currently implemented for the XPU device. Please open a feature on https://github.com/intel/torch-xpu-ops/issues. You can set the environment variable `PYTORCH_ENABLE_XPU_FALLBACK=1` to use the CPU implementation as a fallback for XPU unimplemented operators. WARNING: this will bring unexpected performance compared with running natively on XPU.
FAILED language/test_core.py::test_full[shape0-uint16] - RuntimeError: "eq_xpu" not implemented for 'UInt16'
FAILED language/test_core.py::test_full[shape0-uint32] - RuntimeError: "eq_xpu" not implemented for 'UInt32'
FAILED language/test_core.py::test_full[shape0-uint64] - RuntimeError: "eq_xpu" not implemented for 'UInt64'
FAILED language/test_core.py::test_full[shape1-uint16] - RuntimeError: "eq_xpu" not implemented for 'UInt16'
FAILED language/test_core.py::test_full[shape1-uint32] - RuntimeError: "eq_xpu" not implemented for 'UInt32'
FAILED language/test_core.py::test_full[shape1-uint64] - RuntimeError: "eq_xpu" not implemented for 'UInt64'
FAILED language/test_core.py::test_full[shape2-uint16] - RuntimeError: "eq_xpu" not implemented for 'UInt16'
FAILED language/test_core.py::test_full[shape2-uint32] - RuntimeError: "eq_xpu" not implemented for 'UInt32'
FAILED language/test_core.py::test_full[shape2-uint64] - RuntimeError: "eq_xpu" not implemented for 'UInt64'
```
Local:
```
» pip freeze | grep torch                                                                                                                                                                                                 
-e git+https://github.com/alexbaden/pytorch.git@75f64e12030dfa6f621f1ec2b207892cf8660cdd#egg=torch
```
```
python/test/unit/language/test_core.py::test_full[shape0-int8] PASSED                                                                                                                                                                                                                     [  2%]
python/test/unit/language/test_core.py::test_full[shape0-int16] PASSED                                                                                                                                                                                                                    [  5%]
python/test/unit/language/test_core.py::test_full[shape0-int32] PASSED                                                                                                                                                                                                                    [  8%]
python/test/unit/language/test_core.py::test_full[shape0-int64] PASSED                                                                                                                                                                                                                    [ 11%]
python/test/unit/language/test_core.py::test_full[shape0-uint8] PASSED                                                                                                                                                                                                                    [ 13%]
python/test/unit/language/test_core.py::test_full[shape0-uint16] PASSED                                                                                                                                                                                                                   [ 16%]
python/test/unit/language/test_core.py::test_full[shape0-uint32] PASSED                                                                                                                                                                                                                   [ 19%]
python/test/unit/language/test_core.py::test_full[shape0-uint64] PASSED                                                                                                                                                                                                                   [ 22%]
python/test/unit/language/test_core.py::test_full[shape0-float16] PASSED                                                                                                                                                                                                                  [ 25%]
python/test/unit/language/test_core.py::test_full[shape0-float32] PASSED                                                                                                                                                                                                                  [ 27%]
python/test/unit/language/test_core.py::test_full[shape0-float64] PASSED                                                                                                                                                                                                                  [ 30%]
python/test/unit/language/test_core.py::test_full[shape0-bfloat16] PASSED                                                                                                                                                                                                                 [ 33%]
python/test/unit/language/test_core.py::test_full[shape1-int8] PASSED                                                                                                                                                                                                                     [ 36%]
python/test/unit/language/test_core.py::test_full[shape1-int16] PASSED                                                                                                                                                                                                                    [ 38%]
python/test/unit/language/test_core.py::test_full[shape1-int32] PASSED                                                                                                                                                                                                                    [ 41%]
python/test/unit/language/test_core.py::test_full[shape1-int64] PASSED                                                                                                                                                                                                                    [ 44%]
python/test/unit/language/test_core.py::test_full[shape1-uint8] PASSED                                                                                                                                                                                                                    [ 47%]
python/test/unit/language/test_core.py::test_full[shape1-uint16] PASSED                                                                                                                                                                                                                   [ 50%]
python/test/unit/language/test_core.py::test_full[shape1-uint32] PASSED                                                                                                                                                                                                                   [ 52%]
python/test/unit/language/test_core.py::test_full[shape1-uint64] PASSED                                                                                                                                                                                                                   [ 55%]
python/test/unit/language/test_core.py::test_full[shape1-float16] PASSED                                                                                                                                                                                                                  [ 58%]
python/test/unit/language/test_core.py::test_full[shape1-float32] PASSED                                                                                                                                                                                                                  [ 61%]
python/test/unit/language/test_core.py::test_full[shape1-float64] PASSED                                                                                                                                                                                                                  [ 63%]
python/test/unit/language/test_core.py::test_full[shape1-bfloat16] PASSED                                                                                                                                                                                                                 [ 66%]
python/test/unit/language/test_core.py::test_full[shape2-int8] PASSED                                                                                                                                                                                                                     [ 69%]
python/test/unit/language/test_core.py::test_full[shape2-int16] PASSED                                                                                                                                                                                                                    [ 72%]
python/test/unit/language/test_core.py::test_full[shape2-int32] PASSED                                                                                                                                                                                                                    [ 75%]
python/test/unit/language/test_core.py::test_full[shape2-int64] PASSED                                                                                                                                                                                                                    [ 77%]
python/test/unit/language/test_core.py::test_full[shape2-uint8] PASSED                                                                                                                                                                                                                    [ 80%]
python/test/unit/language/test_core.py::test_full[shape2-uint16] PASSED                                                                                                                                                                                                                   [ 83%]
python/test/unit/language/test_core.py::test_full[shape2-uint32] PASSED                                                                                                                                                                                                                   [ 86%]
python/test/unit/language/test_core.py::test_full[shape2-uint64] PASSED                                                                                                                                                                                                                   [ 88%]
python/test/unit/language/test_core.py::test_full[shape2-float16] PASSED                                                                                                                                                                                                                  [ 91%]
python/test/unit/language/test_core.py::test_full[shape2-float32] PASSED                                                                                                                                                                                                                  [ 94%]
python/test/unit/language/test_core.py::test_full[shape2-float64] PASSED                                                                                                                                                                                                                  [ 97%]
python/test/unit/language/test_core.py::test_full[shape2-bfloat16] PASSED                                                                                                                                                                                                                 [100%]

============================================================================================================================= 36 passed, 10882 deselected in 10.13s =============================================================================================================================
```

It looks like the PyTorch main commit used is quite old: https://github.com/pytorch/pytorch/commit/6c503f1dbbf9ef1bf99f19f0048c287f419df600

So I tried another run with the current main, but it is failing with the same issue: https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/9745777583/job/26894544049

--- a/.github/actions/setup-pytorch/action.yml
+++ b/.github/actions/setup-pytorch/action.yml
@@ -68,6 +68,12 @@ runs:
         sudo mkdir -p /opt/intel
         sudo ln -sfT ${{ inputs.oneapi }} /opt/intel/oneapi
 
+    - name: Generate PyTorch cache key
+      shell: bash
+      run: |
+        PYTORCH_CACHE_KEY=$(echo $PYTHON_VERSION $PYTORCH_COMMIT_ID ${{ hashFiles('scripts/patch-pytorch.sh') }} | sha256sum - | cut -d\  -f1)
+        echo "PYTORCH_CACHE_KEY=$PYTORCH_CACHE_KEY" | tee -a "$GITHUB_ENV"
+
     - name: Load PyTorch from a cache
       id: pytorch-cache
       uses: ./.github/actions/load
@@ -76,7 +82,7 @@ runs:
         CACHE_NUMBER: 11
       with:
         path: pytorch
-        key: pytorch-$PYTHON_VERSION-$PYTORCH_COMMIT_ID-$CACHE_NUMBER
+        key: pytorch-$PYTORCH_CACHE_KEY-$CACHE_NUMBER
 
     - name: Clone PyTorch repository
       if: ${{ steps.pytorch-cache.outputs.status == 'miss' }}

--- a/.github/actions/setup-triton/action.yml
+++ b/.github/actions/setup-triton/action.yml
@@ -1,7 +1,7 @@
 # Builds and installs Triton. Uses git clone in the current directory.
 # Sets the following environment variables:
 # * LLVM_COMMIT_ID
-description: Build and install IPEX wheels
+description: Build and install Triton
 inputs:
   build_llvm:
     description: Build LLVM

--- a/.github/workflows/auto-update-translator-cid.yml
+++ b/.github/workflows/auto-update-translator-cid.yml
@@ -49,16 +49,11 @@ jobs:
         with:
           python-version: '3.10'
 
-      - name: Setup PyTorch with IPEX
+      - name: Setup PyTorch
         if: ${{ env.TARGET_PRID == null }}
         uses: ./.github/actions/setup-pytorch
         with:
-          repository: Stonepia/pytorch
-          ref: ""
-
-      - name: Setup IPEX
-        if: ${{ env.TARGET_PRID == null }}
-        uses: ./.github/actions/setup-ipex
+          repository: pytorch/pytorch
 
       - name: Install test dependencies
         if: ${{ env.TARGET_PRID == null }}

--- a/.github/workflows/build-test-a770.yml
+++ b/.github/workflows/build-test-a770.yml
@@ -8,15 +8,6 @@ on:
         description: Runner label, keep empty for default
         type: string
         default: ""
-      install_ipex:
-        # This boolean parameter defines what PyTorch will be used in the workflow:
-        #   true: Stonepia/pytorch (fork) with IPEX
-        #   false: pytorch/pytorch (upstream) without IPX
-        # In both cases, pytorch_ref below allows specifying a branch, tag, or commit id in the
-        # corresponding repository. If not specified, a default (pinned) version will be used.
-        description: Install Intel PyTorch Extension
-        type: boolean
-        default: false
       pytorch_ref:
         description: PyTorch ref, keep empty for default
         type: string
@@ -54,7 +45,6 @@ jobs:
     with:
       device: a770
       runner_label: ${{ inputs.runner_label }}
-      install_ipex: ${{ inputs.install_ipex }}
       pytorch_ref: ${{ inputs.pytorch_ref }}
       python_version: ${{ matrix.python }}
       upload_test_reports: ${{ inputs.upload_test_reports }}

--- a/.github/workflows/build-test-python.yml
+++ b/.github/workflows/build-test-python.yml
@@ -8,15 +8,6 @@ on:
         description: Runner label, keep empty for default
         type: string
         default: ""
-      install_ipex:
-        # This boolean parameter defines what PyTorch will be used in the workflow:
-        #   true: Stonepia/pytorch (fork) with IPEX
-        #   false: pytorch/pytorch (upstream) without IPX
-        # In both cases, pytorch_ref below allows specifying a branch, tag, or commit id in the
-        # corresponding repository. If not specified, a default (pinned) version will be used.
-        description: Install Intel PyTorch Extension
-        type: boolean
-        default: false
       pytorch_ref:
         description: PyTorch ref, keep empty for default
         type: string
@@ -81,7 +72,6 @@ jobs:
     with:
       driver_version: ${{ matrix.driver }}
       runner_label: ${{ inputs.runner_label }}
-      install_ipex: ${{ inputs.install_ipex }}
       pytorch_ref: ${{ inputs.pytorch_ref }}
       python_version: ${{ matrix.python }}
       upload_test_reports: ${{ inputs.upload_test_reports }}

--- a/.github/workflows/build-test-reusable.yml
+++ b/.github/workflows/build-test-reusable.yml
@@ -1,5 +1,5 @@
 name: Build and test reusable workflow
-run-name: ${{ inputs.run_name }} - ${{ inputs.python_version }} - ${{ inputs.install_ipex && 'IPEX' || 'no IPEX' }} - ${{ inputs.runner_label || 'default'}}
+run-name: ${{ inputs.run_name }} - ${{ inputs.python_version }} - ${{ inputs.runner_label || 'default'}}
 
 on:
   workflow_call:
@@ -16,15 +16,6 @@ on:
         description: Runner label, keep empty for default
         type: string
         default: ""
-      install_ipex:
-        # This boolean parameter defines what PyTorch will be used in the workflow:
-        #   true: Stonepia/pytorch (fork) with IPEX
-        #   false: pytorch/pytorch (upstream) without IPX
-        # In both cases, pytorch_ref below allows specifying a branch, tag, or commit id in the
-        # corresponding repository. If not specified, a default (pinned) version will be used.
-        description: Install Intel PyTorch Extension
-        type: boolean
-        default: true
       pytorch_ref:
         description: PyTorch ref, keep empty for default
         type: string
@@ -100,27 +91,11 @@ jobs:
         with:
           python-version: ${{ inputs.python_version }}
 
-      - name: Setup PyTorch with IPEX
-        if: ${{ inputs.install_ipex }}
-        uses: ./.github/actions/setup-pytorch
-        with:
-          repository: Stonepia/pytorch
-          ref: ${{ inputs.pytorch_ref }}
-
-      - name: Setup PyTorch without IPEX
-        if: ${{ !inputs.install_ipex }}
+      - name: Setup PyTorch
         uses: ./.github/actions/setup-pytorch
         with:
           repository: pytorch/pytorch
           ref: ${{ inputs.pytorch_ref }}
-
-      - name: Setup IPEX
-        if: ${{ inputs.install_ipex }}
-        uses: ./.github/actions/setup-ipex
-
-      - name: Setup no-op IPEX
-        if: ${{ !inputs.install_ipex }}
-        uses: ./.github/actions/setup-no-op-ipex
 
       - name: Install test dependencies
         run: |

--- a/.github/workflows/build-test.yml
+++ b/.github/workflows/build-test.yml
@@ -8,15 +8,6 @@ on:
         description: Runner label, keep empty for default
         type: string
         default: ""
-      install_ipex:
-        # This boolean parameter defines what PyTorch will be used in the workflow:
-        #   true: Stonepia/pytorch (fork) with IPEX
-        #   false: pytorch/pytorch (upstream) without IPX
-        # In both cases, pytorch_ref below allows specifying a branch, tag, or commit id in the
-        # corresponding repository. If not specified, a default (pinned) version will be used.
-        description: Install Intel PyTorch Extension
-        type: boolean
-        default: false
       pytorch_ref:
         description: PyTorch ref, keep empty for default
         type: string
@@ -67,7 +58,7 @@ jobs:
           CACHE_NUMBER: 2
         with:
           path: $HOME/.cache/pip
-          key: pip-3.10-${{ hashFiles('.pre-commit-config.yaml') }}-${{ env.INSTALL_IPEX }}-${{ env.CACHE_NUMBER }}
+          key: pip-3.10-${{ hashFiles('.pre-commit-config.yaml') }}-${{ env.CACHE_NUMBER }}
 
       - name: Install Python 3.10
         uses: actions/setup-python@v5
@@ -128,7 +119,6 @@ jobs:
     with:
       driver_version: ${{ matrix.driver }}
       runner_label: ${{ inputs.runner_label }}
-      install_ipex: ${{ inputs.install_ipex || false }}
       pytorch_ref: ${{ inputs.pytorch_ref }}
       python_version: ${{ matrix.python }}
       upload_test_reports: ${{ inputs.upload_test_reports || false }}

--- a/.github/workflows/e2e-accuracy.yml
+++ b/.github/workflows/e2e-accuracy.yml
@@ -4,10 +4,6 @@ run-name: ${{ inputs.run_name }}
 on:
   workflow_dispatch:
     inputs:
-      install_ipex:
-        description: Install Intel PyTorch Extension
-        type: boolean
-        default: false
       pytorch_ref:
         description: PyTorch ref, keep empty for default
         type: string
@@ -123,7 +119,6 @@ jobs:
       fail-fast: false
     uses: ./.github/workflows/e2e-reusable.yml
     with:
-      install_ipex: ${{ inputs.install_ipex }}
       pytorch_ref: ${{ inputs.pytorch_ref }}
       suite: ${{ matrix.suite }}
       mode: ${{ matrix.mode }}

--- a/.github/workflows/e2e-performance.yml
+++ b/.github/workflows/e2e-performance.yml
@@ -4,10 +4,6 @@ run-name: ${{ inputs.run_name }}
 on:
   workflow_dispatch:
     inputs:
-      install_ipex:
-        description: Install Intel PyTorch Extension
-        type: boolean
-        default: false
       pytorch_ref:
         description: PyTorch ref, keep empty for default
         type: string
@@ -133,7 +129,6 @@ jobs:
       fail-fast: false
     uses: ./.github/workflows/e2e-reusable.yml
     with:
-      install_ipex: ${{ inputs.install_ipex }}
       pytorch_ref: ${{ inputs.pytorch_ref }}
       suite: ${{ matrix.suite }}
       mode: ${{ matrix.mode }}

--- a/.github/workflows/e2e-reusable.yml
+++ b/.github/workflows/e2e-reusable.yml
@@ -3,10 +3,6 @@ name: E2E reusable workflow
 on:
   workflow_call:
     inputs:
-      install_ipex:
-        description: Install Intel PyTorch Extension
-        type: boolean
-        default: false
       pytorch_ref:
         description: PyTorch ref, keep empty for default
         type: string
@@ -87,40 +83,17 @@ jobs:
         run: |
           pip install wheel
 
-      - name: Setup PyTorch with IPEX
-        if: ${{ inputs.install_ipex }}
-        uses: ./.github/actions/setup-pytorch
-        with:
-          repository: Stonepia/pytorch
-          ref: ${{ inputs.pytorch_ref }}
-
-      - name: Setup PyTorch without IPEX
-        if: ${{ !inputs.install_ipex }}
+      - name: Setup PyTorch
         uses: ./.github/actions/setup-pytorch
         with:
           repository: pytorch/pytorch
           ref: ${{ inputs.pytorch_ref }}
 
-      - name: Set benchmark repository (IPEX)
-        if: ${{ inputs.install_ipex }}
-        run: |
-          echo "BENCHMARK_REPO=weishi-deng/benchmark" | tee -a "$GITHUB_ENV"
-
-      - name: Set benchmark repository (PyTorch)
-        if: ${{ !inputs.install_ipex }}
+      - name: Set benchmark repository
         run: |
           echo "BENCHMARK_REPO=pytorch/benchmark" | tee -a "$GITHUB_ENV"
 
-      - name: Get benchmark commit id (IPEX)
-        if: ${{ inputs.install_ipex }}
-        uses: ./.github/actions/get-commit-id
-        with:
-          repository: ${{ env.BENCHMARK_REPO }}
-          branch: main
-          variable: BENCHMARK_COMMIT_ID
-
-      - name: Get benchmark commit id (PyTorch)
-        if: ${{ !inputs.install_ipex }}
+      - name: Get benchmark commit id
         run: |
           cd pytorch
           echo "BENCHMARK_COMMIT_ID=$(<.github/ci_commit_pins/torchbench.txt)" | tee -a "$GITHUB_ENV"
@@ -133,14 +106,6 @@ jobs:
           echo "TORCHAUDIO_COMMIT_ID=$(<.github/ci_commit_pins/audio.txt)" >> "${GITHUB_ENV}"
           echo "TRANSFORMERS_VERSION=$(<.ci/docker/ci_commit_pins/huggingface.txt)" >> "${GITHUB_ENV}"
           echo "TIMM_COMMIT_ID=$(<.ci/docker/ci_commit_pins/timm.txt)" >> "${GITHUB_ENV}"
-
-      - name: Setup IPEX
-        if: ${{ inputs.install_ipex }}
-        uses: ./.github/actions/setup-ipex
-
-      - name: Setup no-op IPEX
-        if: ${{ !inputs.install_ipex }}
-        uses: ./.github/actions/setup-no-op-ipex
 
       - name: Generate Triton cache key
         id: triton-key
@@ -194,12 +159,6 @@ jobs:
           repository: pytorch/vision
           ref: ${{ env.TORCHVISION_COMMIT_ID }}
           extra-cache-key: ${{ env.PYTORCH_VERSION }}
-
-      # FIXME: old PyTorch does not work with torchdata=0.8.0
-      - name: Install torchdata package
-        if: ${{ inputs.suite == 'torchbench' && inputs.install_ipex }}
-        run: |
-          pip install 'torchdata<0.8.0'
 
       - name: Install torchtext package
         if: ${{ inputs.suite == 'torchbench' }}
@@ -290,9 +249,6 @@ jobs:
           PYTORCH_REPO=$PYTORCH_REPO
           PYTORCH_COMMIT_ID=$PYTORCH_COMMIT_ID
           PYTORCH_VERSION=$PYTORCH_VERSION
-          IPEX_REPO=$IPEX_REPO
-          IPEX_COMMIT_ID=$IPEX_COMMIT_ID
-          IPEX_VERSION=$IPEX_VERSION
           LLVM_REPO=llvm/llvm-project
           LLVM_COMMIT_ID=$LLVM_COMMIT_ID
           BENCHMARK_REPO=$BENCHMARK_REPO

--- a/.github/workflows/nightly-wheels.yml
+++ b/.github/workflows/nightly-wheels.yml
@@ -24,8 +24,6 @@ jobs:
           - "3.10"
           - "3.11"
           - "3.12"
-        mode:
-          - pytorch
       fail-fast: false
       max-parallel: 2
     defaults:
@@ -44,14 +42,7 @@ jobs:
         run: |
           pip install wheel
 
-      - name: Setup PyTorch with IPEX
-        if: ${{ matrix.mode == 'ipex' }}
-        uses: ./.github/actions/setup-pytorch
-        with:
-          repository: Stonepia/pytorch
-
-      - name: Setup PyTorch without IPEX
-        if: ${{ matrix.mode == 'pytorch' }}
+      - name: Setup PyTorch
         uses: ./.github/actions/setup-pytorch
         with:
           repository: pytorch/pytorch
@@ -62,10 +53,6 @@ jobs:
           echo "TORCHVISION_COMMIT_ID=$(<.github/ci_commit_pins/vision.txt)" >> $GITHUB_ENV
           echo "TORCHTEXT_COMMIT_ID=$(<.github/ci_commit_pins/text.txt)" >> $GITHUB_ENV
           echo "TORCHAUDIO_COMMIT_ID=$(<.github/ci_commit_pins/audio.txt)" >> $GITHUB_ENV
-
-      - name: Setup IPEX
-        if: ${{ matrix.mode == 'ipex' }}
-        uses: ./.github/actions/setup-ipex
 
       - name: Identify Triton commit id
         run: |
@@ -108,12 +95,6 @@ jobs:
           ref: ${{ env.TORCHVISION_COMMIT_ID }}
           extra-cache-key: ${{ env.PYTORCH_VERSION }}
 
-      # FIXME: old PyTorch does not work with torchdata=0.8.0
-      - name: Install torchdata package
-        if: ${{ matrix.mode == 'ipex' }}
-        run: |
-          pip install 'torchdata<0.8.0'
-
       - name: Install torchtext package
         uses: ./.github/actions/install-dependency
         with:
@@ -139,12 +120,6 @@ jobs:
           cp -L torchtext*/dist/*.whl wheels/
           cp -L torchaudio*/dist/*.whl wheels/
 
-      - name: Prepare IPEX wheels for upload
-        if: ${{ matrix.mode == 'ipex' }}
-        run: |
-          mkdir -p wheels
-          cp -L intel-extension-for-pytorch/dist/*.whl wheels/
-
       - name: Report environment details
         run: |
           TIMESTAMP=$(date '+%Y%m%d')
@@ -158,8 +133,6 @@ jobs:
           PYTHON_VERSION=$PYTHON_VERSION
           PYTORCH_REPO=$PYTORCH_REPO
           PYTORCH_COMMIT_ID=$PYTORCH_COMMIT_ID
-          IPEX_REPO=$IPEX_REPO
-          IPEX_COMMIT_ID=$IPEX_COMMIT_ID
           LLVM_REPO=llvm/llvm-project
           LLVM_COMMIT_ID=$LLVM_COMMIT_ID
           TRITON_REPO=intel/intel-xpu-backend-for-triton
@@ -169,15 +142,7 @@ jobs:
           TORCHAUDIO_COMMIT_ID=$TORCHAUDIO_COMMIT_ID
           EOF
 
-      - name: Upload IPEX wheels to artifacts
-        if: ${{ matrix.mode == 'ipex' }}
-        uses: actions/upload-artifact@v4
-        with:
-          name: wheels-py${{ matrix.python }}-${{ env.TIMESTAMP }}
-          path: wheels
-
       - name: Upload PyTorch wheels to artifacts
-        if: ${{ matrix.mode == 'pytorch' }}
         uses: actions/upload-artifact@v4
         with:
           name: wheels-pytorch-py${{ matrix.python }}-${{ env.TIMESTAMP }}

--- a/.github/workflows/try-latest-pytorch.yml
+++ b/.github/workflows/try-latest-pytorch.yml
@@ -85,7 +85,6 @@ jobs:
     with:
       driver_version: ${{ matrix.driver }}
       runner_label: ${{ inputs.runner_label }}
-      install_ipex: false
       pytorch_ref: ${{ needs.prepare.outputs.pytorch-commit-id }}
       python_version: ${{ matrix.python }}
       upload_test_reports: ${{ inputs.upload_test_reports || false }}

--- a/scripts/pass_rate.py
+++ b/scripts/pass_rate.py
@@ -162,7 +162,6 @@ def print_json_stats(stats: List[ReportStats]):
         'gpu_device': os.getenv('GPU_DEVICE', ''),
         'python_version': platform.python_version(),
         'pytorch_version': os.getenv('PYTORCH_VERSION', ''),
-        'ipex_version': os.getenv('IPEX_VERSION', ''),
         'testsuite': overall.name,
         'passed': overall.passed,
         'failed': overall.failed,

